db/migrate/20190322040530_create_session_activities.rb

```diff
@@ -0,0 +1,21 @@
+class CreateSessionActivities < ActiveRecord::Migration
+  def change
+    create_table :session_activities do |t|
+      t.uuid     :originating_request_id, null: false
+      t.string   :originating_ip_address, null: false
+      t.string   :originating_user_agent
+      t.string   :generated_url, null: false
```

Comment:
generated_url is going to be the string generated from URLService. I suppose its possible if RelayState and other attributes push it over the 255. :/

Reply:
i changed this to text, although it looks like as of Rails 4, it's treated as a text automatically since postgresql apparently has no performance differences between varchar and blob?